config.ru

```diff
@@ -1,2 +1,7 @@
+#\ --quiet
```

Comment:
Is this functionality that could be folded into the gem at some point? Or are we aiming to keep the gem server-agnostic?

Reply:
I've raised it as an issue on sensible_logging to see if we can override rack: https://github.com/madetech/sensible_logging/issues/24

If we can we'll update the code in these APIs accordingly.